webapp/src/Controller/API/AwardsController.php

```diff
@@ -197,11 +188,11 @@ protected function getAwardsData(Request $request, string $requestedType = null)
 
     protected function getQueryBuilder(Request $request): QueryBuilder
     {
-        throw new NotImplementedException();
+        throw new Exception('Not implemented');
```

Comment:
how is that better?

Reply:
NotImplementedException was deprecated. Also that error was for 'you didnt implement I18n' which was totally stupid.